### PR TITLE
Add possibility to filter for tag

### PR DESCRIPTION
**Pull request by cmichi**
I had the use case of having to restrict urlwatch to the body and this is the way I realized it.

This PR would also solve #155.

### CODE REVIEW

**Comment by cmichi** [comment]
Good point. I have amended the PR. Can you please take a look again?

File: lib/urlwatch/filters.py [hunk 1]
```diff
@@ -217,11 +219,21 @@ def filter(self, data, subfilter=None):
         return data.strip()
 
 
-class ElementsByAttribute(html.parser.HTMLParser):
-    def __init__(self, name, value):
+class FilterBy(Enum):
+    attribute = 1
```
Comment (thp) [inline lib/urlwatch/filters.py]: Make those named constants uppercase (`ATTRIBUTE = 1` and `TAG = 2`)

File: lib/urlwatch/filters.py [hunk 2]
```diff
@@ -217,11 +219,21 @@ def filter(self, data, subfilter=None):
...
+    tag = 2
+
+
+class ElementsBy(html.parser.HTMLParser):
+    def __init__(self, filterBy, name, value=None):
```
Comment (thp) [inline lib/urlwatch/filters.py]: PEP-8: `filterBy` -> `filter_by`

**Comment by cmichi** [comment]
Alright, I have updated the PR with your comments.

**Comment by cmichi** [comment]
Thank you as well! You could close #155 now too.

**Pull request merged**